ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionViewModel.kt

```diff
@@ -123,9 +123,11 @@ internal constructor(
 
   fun loadSubmissionDetails(args: DataCollectionFragmentArgs) = argsProcessor.onNext(args)
 
-  fun getTaskViewModel(position: Int, task: Task): AbstractTaskViewModel {
+  fun getTaskViewModel(position: Int): AbstractTaskViewModel {
     val viewModels = taskViewModels.value
     require(viewModels != null)
+    val tasks = submission.value!!.job.tasksSorted
```

Comment:
How much work would it be to do this without the `!!`? 

If it's null we should probably handle it with some kind of error toast or screen. Maybe let's add a `require` now and TODO for handling this case?

Reply:
Updated to requireNotNull. I think it would probably be a decent amount of work do without that.